.github/workflows/main.yml

```diff
@@ -71,6 +71,7 @@ jobs:
         uses: chromaui/action@v1
```

Comment:
Can you make some tasks (I guess tech debt?) to address the other points Jesse made:
- re-export tag names from nimble-angular so clients don't have to depend on both it and nimble-components
- update existing usages of DesignSystem.tagFor() [in client code](https://dev.azure.com/ni/DevCentral/_search?action=contents&text=designsystem.tagfor&type=code&lp=code-Project&filters=ProjectFilters%7BDevCentral%7DRepositoryFilters%7BSkyline%7D&pageSize=25&result=DefaultCollection/DevCentral/Skyline/GBmaster//Web/Workspaces/SystemLinkShared/projects/systemlink-lib-angular/src/sl-dialog/sl-dialog-service.ts) to use the new pattern
- possibly documenting the API in README or storybook

We should also use the new tag names consistently in nimble instead of DesignSystem.tagFor where it makes sense (i.e. not for injecting the component registered to a base class but for when tags are used directly).

The above shouldn't be for this PR, this one is big enough.

Reply:
Looks like there are quite a few uses of `DesignSystem.tagFor()` in Nimble code and a couple references to it in CONTRIBUTING too: [github search](https://github.com/search?q=repo%3Ani%2Fnimble%20designsystem.tagfor&type=code)

Reply:
Added https://github.com/ni/nimble/issues/1094 & https://github.com/ni/nimble/issues/1095 to track these items - 👍